packages/teams-js/src/public/meeting.ts

```diff
@@ -469,14 +485,18 @@ export namespace meeting {
   }
 
   /**
-   * Allows an app to request the live streaming be started at the given streaming url
+   * Allows an app to request the meeting to start live streaming to the given RTMP streaming url.
```

Comment:
I'm not sure I understand this comment. The way it's worded it kind of sounds like the function is returning a meeting object ("request the meeting to start live streaming to"). Maybe a small change, something like "Allows an app to ask the current meeting to start live streaming to the given RTMP streaming url" would have helped me

Reply:
Hi @TrevorJoelHarris, @ryanbliss I think it's phrased this way because the app cannot start streaming without the user explicitly clicking to allow:

![image](https://github.com/OfficeDev/microsoft-teams-library-js/assets/89165999/ea93a8ad-2e0b-4d17-a783-b8925b5a82a0)

I agree your phrasing is helpful, Trevor, or maybe:

" Allows an app to start live streaming the current meeting to the given RTMP streaming url. A prompt will be shown to the invoking user, before completion, with the option to "Allow" or "Cancel" this request. "

Please let me know what you both think

Reply:
I like it! Updating :)

Reply:
How is this?
```
Allows an app to ask the local user to begin live streaming the current meeting to the given Real-Time Messaging Protocol (RTMP) streaming url.
A confirmation dialog will be shown to the local user with options to "Allow" or "Cancel" this request.
```